trl/trainer/dpo_trainer.py

```diff
@@ -110,11 +112,13 @@ class DPOTrainer(Trainer):
         compute_metrics (`Callable[[EvalPrediction], Dict]`, *optional*):
             The function to use to compute the metrics. Must take a `EvalPrediction` and return
             a dictionary string to metric values.
+        precompute_ref_log_probs (`bool`, defaults to `False`):
```

Comment:
How about raising a `ValueError` in the init if DeepSpeed ZeRO-3 is activated with a message like `"`precompute_ref_log_probs` is not supported with DeepSpeed ZeRO-3."`?